starlette/datastructures.py

```diff
@@ -123,76 +123,76 @@ class Headers(typing.Mapping[str, str]):
     An immutable, case-insensitive multidict.
     """
 
-    def __init__(self, raw_headers=None) -> None:
+    def __init__(self, raw_headers: typing.Any = None) -> None:
         if raw_headers is None:
-            self._list = []
+            self._list = []  # type: typing.List
```

Comment:
We need to define the type, otherwise we have `error: Need type annotation for '_list'`

In this case should be: `self._list = []  # type: BytePairs`